MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -153,48 +82,26 @@ final class MovieQuizViewController: UIViewController {
         //2) Показть алерт при помощи паттерна "Делегат"
         alertView?.show()
     }
-    private func showLoadingIndicator() {
-        downloadMoviesIndicator.isHidden = false // говорим, что индикатор загрузки не скрыт
-        downloadMoviesIndicator.startAnimating() // включаем анимацию
+    func showLoadingIndicator() {
+        downloadMoviesIndicator?.isHidden = false // говорим, что индикатор загрузки не скрыт
+        downloadMoviesIndicator?.startAnimating() // включаем анимацию
     }
-    private func showNetworkError(message: String) {
-        downloadMoviesIndicator.isHidden = true
+    func showNetworkError(message: String) {
+        downloadMoviesIndicator?.isHidden = true
     
         var alertModel = AlertViewModel(
             title: "Что-то пошло не так",
             message: "Невозможно загрузить данные",
             buttonText: "Попробовать ещё раз"
         )
-        alertModel.handler = { [weak self] _ in
-            guard let self = self else { print("Can't assign proper handler for alertModel"); return}
-        }
+        alertModel.handler = { _ in }
         alertView = AlertPresenter(
                                   delegate: self,
                                   alertSome: alertModel
                                   )
```

Comment:
Такого стиля с переносами строк я не встречала, поэтому предложу более популярный
```suggestion
        alertView = AlertPresenter(
            delegate: self, // аргументы переносятся на новую строку, один таб глубже предыдущей
            alertSome: alertModel
         ) // закрывающая скобка всегда вровень с открывающей строкой (alertView =)
```

Reply:
Буду так оформлять список параметров со значениями, начиная со следующего проекта. 
Спасибо за внимание!